src/core.ts

```diff
@@ -358,6 +360,20 @@ function prepare() {
   return true;
 }
 
+function sendClientInfo() {
```

Comment:
>sendClientInfo [](start = 9, length = 14)

PageData seems better to me for a name.

Reply:
It also contains clientId - let's call it metadata

---
In reply to: [144391024](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144391024) [](ancestors = 144391024)